src/style.css

```diff
@@ -0,0 +1,193 @@
+* {
```

Comment:
Optional, I would suggest setting text transform to capitalise as the user can enter words awkwardly

Reply:
You mean to make all text in entire page capitalized?